pkg/cmd/log.go

```diff
@@ -0,0 +1,223 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
+)
+
+type LogOptions struct {
+	ContainerName string
+	Name          string
+	Resource      string
+	GroupVersion  string
+	LabelSelector string
+
+	IsCoreResource bool
```

Comment:
Even if this is ok considering how the `get` command works, I would want to see how do you think about removing this parameter and getting the cli to know if the resource is core or not given the groupversion.

What do you think?
I can open an issue for it